bitsandbytes/cuda_setup/main.py

```diff
@@ -31,7 +32,13 @@
 # libcudart.so is missing by default for a conda install with PyTorch 2.0 and instead
 # we have libcudart.so.11.0 which causes a lot of errors before
 # not sure if libcudart.so.12.0 exists in pytorch installs, but it does not hurt
-CUDA_RUNTIME_LIBS: list = ["libcudart.so", 'libcudart.so.11.0', 'libcudart.so.12.0']
+system = platform.system()
+if system == 'Darwin':
+    CUDA_RUNTIME_LIBS: list = ["libcuda.dylib", '/usr/local/cuda/lib/libcuda.dylib']
```

Comment:
Can you explain to me a bit where this very fixed path on Drawin comes from?

Reply:
oops. I guess this line is an outdated hunk from some older cuda detection routine and not cleaned up before posting this PR.